### ISSUE TITLE
Small wall-castle transition issue

### ISSUE THREAD

**Issue by Vultraz** [wesnoth/wesnoth#2190]
See below:
![transition issue](https://user-images.githubusercontent.com/3558659/32695679-7289c928-c7b6-11e7-9279-72b5a6b543fa.PNG)

@doofus-01 @ln-zookeeper perhaps either of you know how this could be fixed? The brazier makes no difference, BTW.

**Comment by doofus-01** [comment]
I think I know what the problem is, though the fix might cause trouble...  It's like one of those sliding tile puzzles.  If zookeeper doesn't get to it, I'll try to have something within the week.

**Comment by doofus-01** [comment]
It's a problem with base key and rotations.  Unfortunately, changing the base so that the wall corner image is above the castle wall results in other layering glitches.
I can change the base key at line 2483 of new-macros.cfg to get the glitch to go away, but it might cause other issues.  I don't see any though, so I'll just go ahead and do that...

**Comment by Vultraz** [comment]
I'm not seeing any difference even with your change :/

**Comment by doofus-01** [comment]
Yes, seems like that didn't catch all cases.

**Comment by doofus-01** [comment]
Try it now?

**Comment by Vultraz** [comment]
Confirmed fixed \o/

**Comment by CelticMinstrel** [comment]
@doofus-01 – The issue got closed because your commit message contained the string "fix #2190". If you're not sure the fix works and you want to leave the issue open, you should either use a different word (eg something like "partially addresses #2190") or rephrase it to place something between (eg "partial fix for #2190").
